libfwupd/fwupd-client.c

```diff
@@ -2112,6 +2219,10 @@ fwupd_client_install_stream_async (FwupdClient *self,
  *
  * Install firmware onto a specific device.
  *
+ * NOTE: This method is thread-safe, but progress signals will be
+ * emitted in the default main context, if not explicitly set with
```

Comment:
Would be good to clarify what you mean by ‘default main context’. Do you mean the global default main context (`g_main_context_default()`), the thread default main context (`g_main_context_get_thread_default()`) or something else specific to `FwupdClient`?

Reply:
> Do you mean the global default main context

Exactly that, will change to that wording.